Documentation/5.4.1/README.rst

```diff
@@ -306,17 +306,23 @@ Phase 2: Modify the Overcloud qcow image (for example, overcloud-full.qcow2) to
 The steps for modifying overcloud-full.qcow2 are provided in the `README.md <../../image-patching/stopgap-script/README.md>`_ file.
 
 
-Phase 3: Update the Undercloud codebase.
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Phase 3: Adding Nuage Heat Templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Copy the nuage-tripleo-heat-templates folder form `here <../../nuage-tripleo-heat-templates>`_ to /home/stack/ directory on undercloud.
```

Comment:
Rather than copy, maybe something like this:

    cd /home/stack
    git clone (base repository) nuage-ospdirector
    ln -s nuage-ospdirector/nuage-tripleo-heat-templates .

This means that changes are made in the git directory itself, and can be checked in to a new branch for tracking more easily.

Reply:
done